src/pages/People.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Table } from '../components/Table';
+
+const peopleToRender = (people: Person[]) => {
```

Comment:
```suggestion
const preparePeople = (people: Person[]) => {
```
A function name should start with a verb, and you can move this function to utils/helpers